cartesian_trajectory_controller/CMakeLists.txt

```diff
@@ -203,6 +203,14 @@ install(FILES
   ${PROJECT_NAME}_plugin.xml
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
+# Install test setup files for packages that re-use them
```

Comment:
Wouldn't it make more sense to define a separate test package that can be used by different packages?

Reply:
... I'm hesitating. It's really not much files and they fit well into the cartesian_trajectory_controller package.